src/utilities/https-helper.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+/**
+ * 
+ * @returns array which contains video data
+ */
```

Comment:
suggestion: add one-line description of what this function does

Reply:
added